webui/src/components/reportabuse.jsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react/lib/ReactWithAddons';
+import { Link, browserHistory } from 'react-router'
+
+import Toggle from 'react-toggle';
+import { DropdownList } from 'react-widgets';
+import { HeightAnimate, ReplaceAnimate } from './animate.jsx';
+
+//TODO: handle the submission
```

Comment:
On submission, the javascript client must send a request to the B2SHARE server. We must add a new HTTP API endpoint (either in /records/ or elsewhere) for this. The server would just pack all the data it gets and send an email to the eudat ticketing system. See for reference this old file: https://github.com/EUDAT-B2SHARE/b2share/blob/1.7/invenio/b2share/modules/abuse/abuse_form.py